jobs/rocoto/aeroanlinit.sh

```diff
@@ -1,12 +1,15 @@
 #! /usr/bin/env bash
 
-source "$HOMEgfs/ush/preamble.sh"
+source "${HOMEgfs}/ush/preamble.sh"
 
 ###############################################################
 # Source FV3GFS workflow modules
-. $HOMEgfs/ush/load_fv3gfs_modules.sh
+. "${HOMEgfs}/ush/load_fv3gfs_modules.sh"
```

Comment:
Same comment as for [aeroanlfinal.sh](https://github.com/NOAA-EMC/global-workflow/pull/1106/files#diff-88e6a89b193e77314044c90c9f05e957dc108195d4c40c7b2daa394c56a92a3f)